src/FirmataMarshaller.h

```diff
@@ -0,0 +1,79 @@
+/*
+  FirmataMarshaller.h
+  Copyright (c) 2006-2008 Hans-Christoph Steiner.  All rights reserved.
+  Copyright (C) 2009-2016 Jeff Hoefs.  All rights reserved.
+
+  This library is free software; you can redistribute it and/or
+  modify it under the terms of the GNU Lesser General Public
+  License as published by the Free Software Foundation; either
+  version 2.1 of the License, or (at your option) any later version.
+
+  See file LICENSE.txt for further informations on licensing terms.
+*/
+
+#ifndef FirmataMarshaller_h
+#define FirmataMarshaller_h
+
+#if defined(__cplusplus) && !defined(ARDUINO)
+  #include <cstddef>
+  #include <cstdint>
+#else
+  #include <stddef.h>
+  #include <stdint.h>
+#endif
+
+#ifndef PARTICLE
+#include <Stream.h>
+#else
+#include "Particle.h"
```

Comment:
I don't think this is necessary anymore now that Particle exposes Stream.h in the user space. However if you've tried and encountered errors or need to maintain backwards compatibility then including Particle.h is still necessary.